ndmg/scripts/ndmg_bids.py

```diff
@@ -187,7 +249,9 @@ def main():
                         'will be performed. Multiple participant level '
                         'analyses can be run independently (in parallel) '
                         'using the same output_dir.',
-                        choices=['session', 'participant', 'group'])
+                        choices=['participant', 'group'], default='participant')
+    parser.add_argument('modality', help='Modality of MRI scans that \
```

Comment:
This should be optional, and set as a flag like other optional arguments for consistency. If not specified, check for data for both, and do them each if possible/data is available.

Reply:
just added flag that defaults to dwi for now